target/linux/layerscape/base-files/etc/board.d/02_network

```diff
@@ -14,6 +14,19 @@ case "$(board_name)" in
 		ucidef_set_interface_lan "eth0 eth1 eth2 eth3 eth6"
 		ucidef_set_interface_wan "eth4"
 		;;
+	fsl,ls1028a-rdb*)
```

Comment:
Please don't use wildcards for board names in base-files. This makes it impossible to grep for a specific device string.

Reply:
Ok.